plugins/modules/cs_instance.py

```diff
@@ -135,6 +135,18 @@
       - Only considered when I(state=started) or instance is running.
       - Requires root admin privileges.
     type: str
+  cluster:
+    description:
+      - Cluster on which an instance should be deployed or started on.
+      - Only considered when I(state=started) or instance is running.
+      - Requires root admin privileges.
+    type: str
+   pod:
```

Comment:
The logs for the CI run are no longer here, so I'm not sure whether this caused the sanity tests to fail, but the above line is indented wrongly, so this YAML documentation is malformed now.